configure.ac

```diff
@@ -471,7 +471,11 @@ dnl Multi threaded library.
      [dnl Check for IBM's compiler's TLS option.
       AX_CXXFLAGS_AIX_OPTION([-qtls])])
 
-   AX_TLS_SUPPORT
+   dnl Check if TLS is supported, but only if it isn't iOS target, because it
+   dnl is forbidden by Apple Store to use __tlv_* functions.
+   AS_IF([test "$target_os" != "darwin" -o "$target_cpu" != "arm",
```

Comment:
Just FYI, I have caught a bug in this change: The `[]` are wrongly balanced. This is fixed by 7b5f53d26f8b2835fb08a7c72af82d5d69eb3bca. Next time, please run `./scripts/doautoreconf.sh` and inspect the resulting diff as well. :)